tasks/post-to-cmr/tests/cmr-test.js

```diff
@@ -19,7 +19,8 @@ const { isCMRFile } = require('@cumulus/cmrjs');
 const { postToCMR } = require('..');
 
 const result = {
-  'concept-id': 'testingtesting',
+  'concept-id': 'G1222482316-CUMULUS',
+  'revision-id': 123,
```

Comment:
Same comment as above: since I don't think we really use revision-id from the return, I'm hesitant to include it in the mock